src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -1,10 +1,297 @@
 #cython: language_level=3
 from enum import IntEnum
-from charfbuzz cimport *
 from libc.stdlib cimport free, malloc
 from libc.string cimport const_char
 from typing import Callable, Dict, List, Tuple
 
+from libc.stdint cimport uint8_t, uint16_t, uint32_t
+
+
+cdef extern from "hb.h":
```

Comment:
Had to paste content of charfbuzz.pxd to this to make it work, thought `from .charfbuzz cimport *` would work but didn't? some help would be nice, thanks!

Reply:
`from charfbuzz cimport *` (without the `.`, as it was before) should work, does it not?

Reply:
It doesn't, can you help me and test it locally? The patch is of course just a PoC right now.

Reply:
And not that I don't want fix it but I think at this point that it is a cython and it's unable to properly import things from the same directory and was thinking about moving the file to root of the project (not tested however if that help, will soon) and just thought as you are more familiar with things here and maybe interested on helping making it happen, if not, guess will be able to find my way somehow as it currently builds successfully at least.

Reply:
I think the issue is that you named the Extension just `uharfbuzz`, as standalone module without a package; before that, it was (implicitly) named `uharfbuzz._harfbuzz` (note the `uharfbuzz.` prefix, which is the name of the enclosing package where the extension module is located). Then cython compiler will extend the pxd include paths to also include the package directory (where charfbuzz.pxd is located).
Try to rename the Extension to `uharfbuzz._harfbuzz` and I think the `cimport` should work then.

Reply:
Thank you so much! It works now :)